### DISCUSSION TITLE
Why dims=2:30

### DISCUSSION THREAD

**Discussion by summer-tao** [stuart-lab/signac#573] [Q&A] [answered]
Dear Tim,

I want to know why dims=2:30, discard the first one? I thought the first few components are super important.

pbmc <- RunUMAP(object = pbmc, reduction = 'lsi', **dims = 2:30**)
pbmc <- FindNeighbors(object = pbmc, reduction = 'lsi', **dims = 2:30**)
pbmc <- FindClusters(object = pbmc, verbose = FALSE, algorithm = 3)
DimPlot(object = pbmc, label = TRUE) + NoLegend()

Best,
Tao.

**Comment by timoast** [comment, marked as answer]
The first LSI component is often highly correlated with sequencing depth (technical variation), and so we typically remove it. See https://satijalab.org/signac/articles/pbmc_vignette.html#normalization-and-linear-dimensional-reduction-1

**Reply by summer-tao** [reply]
Thank you, Tim.
